src/libraries/README.md

```diff
@@ -2,22 +2,37 @@
 
 This folder contains the source and tests for the .NET Libraries. Different libraries are owned by different team members; refer to the [Areas](../../docs/area-owners.md#areas) list for lead and owner information.
 
-## Development Statuses
+## Contribution Bar
 
-Some libraries are under more active development than others. Depending on the library's status, expectations for issues and pull requests can vary. Check the library's folder for a `README.md` that declares the status for that library. Regardless of a library's status, refer to the [DOs and DON'Ts](../../CONTRIBUTING.md#dos-and-donts) and [Suggested Workflow](../../CONTRIBUTING.md#suggested-workflow) in our contribution guidelines before submitting a pull request.
+Some libraries are under more active development than others. Depending on the library's status, expectations for issues and pull requests can vary. Check the library's folder for a `README.md` that declares the contribution bar for that library which consists of  a **Primary bar** and optional **Secondary bars**. Regardless of a library's contribution bar, refer to the [DOs and DON'Ts](../../CONTRIBUTING.md#dos-and-donts) and [Suggested Workflow](../../CONTRIBUTING.md#suggested-workflow) in our contribution guidelines before submitting a pull request.
 
-- **Active**
-  - Under active development by the team
-  - Issues will be considered for fix or addition to the backlog
+### Assumed bars (unless a library says otherwise)
+- **We take security fixes**
+- **We take test changes**
+- **We don't take style-only changes**
+
+### Primary bar
```

Comment:
If a library has a different composition, the thinking is it doesn't necessarily need to update this list and doesn't necessarily need to link back to here.